tests/e2e/pageObjects/Ocis.ts

```diff
@@ -0,0 +1,41 @@
+import { Page } from '@playwright/test'
+import { state } from '../hooks'
+import config from '../config'
+import util from 'util'
+
+export class Ocis {
+  static page: Page = state.page
+  static elements: Readonly<Record<string, string>> = {
+    userNameSelector: '#oc-login-username',
+    passwordSelector: '#oc-login-password',
+    loginButtonSelector: 'button[type="submit"]',
+    webContentSelector: '#web-content',
+    userMenuButtonSelector: '#_userMenuButton',
+    logoutSelector: '#oc-topbar-account-logout',
+    resourceUploadButton: '#upload-menu-btn',
+    fileUploadInput: '#files-file-upload-input',
+    uploadInfoCloseButton: '#close-upload-info-btn',
+    resourceNameSelector: '#files-space-table [data-test-resource-name="%s"]'
+  }
+
+  static async login({ username, password }): Promise<void> {
+    await Ocis.page.locator(this.elements.userNameSelector).fill(username)
```

Comment:
looks correct. does it work?

Reply:
nope, I think it has something to do with the fact that "page" is undefined at the time it is called...
see also [comment above](https://github.com/saw-jan/web-app-3dmodel-viewer/pull/12#discussion_r1549225756)

Reply:
I wasn't able to make it work with ocis being a static class